src/deployer-watch/deployer-watch.js

```diff
@@ -3,57 +3,49 @@ const { Finding, FindingSeverity, FindingType } = require('forta-agent');
 const addresses = require('./deployer-watch.json');
 const config = require('../../agent-config.json');
 
-const deployerAddress = addresses.Deployer.toLowerCase();
-const whitelist = {};
+const initializeData = {};
 
-// re-format the whitelist to simplify the logic for checking transactions
-// also remove checksums
-addresses.Whitelist.forEach((a) => {
-  whitelist[a.toLowerCase()] = true;
-});
+function provideInitialize(data) {
+  return async function initialize() {
+    // eslint-disable-next-line no-param-reassign
```

Comment:
Not required (it only saves one line of code), but you could replace the 3 uses of:
`// eslint-disable-next-line no-param-reassign`
with a block disable using this comment at the start:
`/* eslint-disable no-param-reassign */`
and this comment at the end:
`/* eslint-enable no-param-reassign */`